sql-to-dbsp-compiler/README.md

```diff
@@ -90,7 +90,7 @@ See the [documentation](https://www.feldera.com/docs/sql/intro)
 
 ## Compiler architecture
 
-[A presentation](https://github.com/feldera/feldera/blob/main/docs/static/sql-compiler-architecture.pptx)
+[A presentation](https://mihaibudiu.github.io/work/sql-compiler-architecture23.pptx)
```

Comment:
Would be better to point to the feldera URL.

Reply:
there is no feldera URL, I don't know where this file has gone.